### PR TITLE
Update script to terminate native app

### PR DESCRIPTION
**Pull request by khansaad**
Signed-off-by: Saad Khan <saakhan@redhat.com>

### CODE REVIEW

File: scripts/cluster-helpers.sh [hunk 1]
```diff
@@ -94,9 +94,12 @@ function native_terminate() {
 	echo
 	echo -n "###   Stopping HPO Service"
 	echo
-		
+
+	pkill -f service.py >/dev/null 2>&1
+	check_err "Failed to stop HPO Service!"
+
```
Comment (dinogun) [inline scripts/cluster-helpers.sh:100]: Is pkill a standard package that is available on all distros? Why cant we use `kill` here ?

Comment (khansaad) [reply scripts/cluster-helpers.sh]: Updated to 'kill' now

**Review by chandrams — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**